src/Text2Image/T2IParamTypes.cs

```diff
@@ -385,6 +385,9 @@ public static void RegisterDefaults()
         InitImage = Register<Image>(new("Init Image", "Init-image, to edit an image using diffusion.\nThis process is sometimes called 'img2img' or 'Image To Image'.",
             null, OrderPriority: -5, Group: GroupInitImage, ChangeWeight: 2
             ));
+        EditPrompt = Register<string>(new("Edit Prompt", "Prompt to use specifically for editing images.\nThis will override the main prompt when generating edited images.",
+            "", OrderPriority: -4.9, Group: GroupInitImage, ViewType: ParamViewType.PROMPT
```

Comment:
this needs either a toggleable or an ignoreif empty. It should probably also be advanced (the average person doesn't need this).